plugins/survey/src/types.ts

```diff
@@ -0,0 +1,151 @@
+//
+// Copyright © 2023 Hardcore Engineering Inc.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import { AttachedDoc, Class, CollectionSize, Doc, Markup, Ref, Space } from '@hcengineering/core'
+import { Attachment } from '@hcengineering/attachment'
+import { Person } from '@hcengineering/contact'
+import { ComponentType, SvelteComponent } from 'svelte'
+import { Resource } from '@hcengineering/platform'
+
+/**
+ * @public
+ *
+ * @link https://www.npmjs.com/package/lexorank
+ */
+export type Rank = string
```

Comment:
Minor: should it be defined in more common place? It's effectively reused across a lot of various plugins.